lib/improver/nbhood_percentiles.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes. I'm expecting to do a bit of refactoring once all @gavinevans' changes are in.